packages/nodes-base/scripts/watch.mjs

```diff
@@ -0,0 +1,20 @@
+import chokidar from 'chokidar';
+import shell from "shelljs";
+import path from 'path';
+import {fileURLToPath} from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const listener = (event, path) => {
+	shell.exec('npm run build:esbuild');
+}
+
+chokidar.watch([
+	path.resolve(__dirname, '..', 'credentials'),
+	path.resolve(__dirname, '..', 'nodes'),
+	path.resolve(__dirname, '..', 'src'),
+], { ignoreInitial: true })
+	.on('change', listener)
+	.on('add', listener)
+	.on('unlink', listener);
```

Comment:
1. Curious why a chokidar setup instead of [esbuild watch mode](https://esbuild.github.io/api/#watch)? I expect this rebuilds on changes to any file, not just TS files, i.e. more than what we need?
2. Should we rebuild on `unlink`?
3. If `ignoreInitial` is `false`, it glitches out by re-detecting constantly. If `ignoreInitial` is `true`, then building before watching becomes required, i.e. if we watch without building first, then no `/dist` is created until the first change. This differs from the behavior of `tsc --watch`, which knows to build if there is no `/dist`.
4. Nitpick: Most watch tools give out some feedback e.g. `Listening for file changes...`, but this watch script is silent.

Reply:
Good point! Switched to using esbuild in watch mode and added feedback and error handling.